2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 6 Line 2_
```markdown
## Impurity feature importance
Impurity feature importance is a special type of feature importance that is only relevant for random forest models and is sometimes called "gini importance" or "mean decrease impurity". Impurity feature importance is defined as the total decrease in node impurity, weighted by the probability of reaching that node (which is approximated by the proportion of samples reaching that node), averaged over all trees of the ensemble. Simply said, how much does the data in our node remain varied after a split decision (based on a specific input feature) is made.
```
You could mention it is the same that we used last workshop.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._